oauthenticator/google.py

```diff
@@ -105,6 +105,14 @@ def _userdata_url_default(self):
         """,
     )
 
+    allow_nested_groups = Bool(
```

Comment:
Can we make this `include_nested_groups` after all? I've gone back and forth in my mind about this, but I recall you also said you thought it could make sense.

I think my concluding motivation for making this be `include` is that it affects more than the `allowing` part, such as being declared an admin if found in a `admin_google_groups` and if the group is to be listed in auth state, which in turn currently or in the future (i don't remember current status) may be used to influence jupyterhub groups if `manage_groups` is set true.